### ISSUE TITLE
Include README for starred items in private feed

### ISSUE THREAD

**Issue by endquote** [isaacs/github#695]
I subscribe to my "private feed" `https://github.com/endquote.private.atom?token=...` via an RSS reader and enjoy seeing what people I follow are starring, but the items in the feed include only the name of the repo and a link to it. It would be super useful to include the contents of the project's README, even if it were truncated.

**Comment by TPS** [comment]
At least the short project description would probably be useful, if not the full README.

**Comment by endquote** [comment]
Agreed, that would probably be easier to implement. But that's a really short description. I'd take either though!
